internal/const.go

```diff
@@ -8,3 +8,8 @@ package internal // import "go.mongodb.org/mongo-driver/internal"
 
 // Version is the current version of the driver.
 var Version = "local build"
+
+// SetServiceID enables a mode in which the driver mocks server support for returning a "serviceId" field in "hello"
+// command responses by using the value of "topologyVersion.processId". This is used for testing load balancer
+// support until an upstream service can support running behind a load balancer.
+var SetServiceID = false
```

Comment:
Would this be better named `SetMockServiceID`? After 5.0 servers start responding with a `serviceID` in the hello response, I assume the spec will be updated to remove this mock.

Reply:
Renamed.